experimental/tf_proxy/client/src/main.rs

```diff
@@ -24,7 +24,10 @@ use maplit::hashmap;
 use oak_functions_abi::proto::{ConfigurationInfo, Request};
```

Comment:
pre-existing: is it possible to move the imported parts from `oak_functions_abi` to `grpc_attestation`, and use that both here and in `oak_functions`?

Reply:
I created #2495 to track this. I think there are other parts in the crate (and even in `abi.proto`) that is unrelated to the Wasm ABI.

I will do some of the cleanup as part of the PR that will be updating the Oak Functions loader to use the remote attestation proxy.

Reply:
Sounds good. I think it make sense to do some refactoring and even splitting the `abi.proto` file.